source/blog/2017-04-27-rspec-3-6-has-been-released.md

```diff
@@ -0,0 +1,94 @@
+---
+title: RSpec 3.6 has been released!
+author: Sam Phippen, TODO: other editors
+---
+
+RSpec 3.6 has just been released! Given our commitment to
+[semantic versioning](http://semver.org/), this should be a trivial
+upgrade for anyone already using RSpec 3, but if we did introduce
+any regressions, please let us know, and we'll get a patch release
+out with a fix ASAP.
+
+RSpec continues to be a community-driven project with contributors
+from all over the world. This release includes over XXX commits and YYY
+merged pull requests from over 50 different contributors!
+
+Thank you to everyone who helped make this release happen!
+
+## Notable Changes
+
+### Core: Failure counts of errors occurring outside examples
```

Comment:
IMO, the fact that we count failures is less interesting than the fact that we handle these errors at all.  Before the changes in RSpec 3.6 (such as rspec/rspec-core#2323), errors that occurred in `:suite` hooks or while loading spec files would just crash the ruby interpreter and you'd get ruby's default error output including a full stacktrace.  Now we handle these errors and provide nicely formatted output, including backtrace filtering, error-site code snippet, and mentioning the count of these errors in the summary output.

WDYT of this instead?

<hr />

### Core: Errors outside examples now handled and formatted well

In previous versions of RSpec, we allowed errors encountered while loading spec files or running `:suite` hooks to crash the ruby interpreter, giving you its default full-stacktrace output.  In RSpec 3.6, we now handle all errors that occur outside examples, and format them nicely including a filtered backtrace and a code snippet for the site of the error.  For example, an error in a `before(:suite)` hook is now formatted like this:

<img width="479" alt="screen shot 2017-04-27 at 9 59 12 pm" src="https://cloud.githubusercontent.com/assets/49391/25514870/cb9db6c8-2b94-11e7-952f-f26fd783512b.png">

Thanks to Jon Rowe for assisting with this feature.